css/about.css

```diff
@@ -0,0 +1,52 @@
+.about-top{
+    display: flex;
+    flex-direction: column;
+    padding: 0 2%;
```

Comment:
"About" margin/padding doesn't line up with "Strategy" and middle elements on about page